processor.js

```diff
@@ -16,55 +16,76 @@ var notifier_messager = require("./messager/messager")({
   queueUrl: process.env.OUTGOING_QUEUE_URL
 });
 
-var mailroom = require('webmaker-mailroom')();
+var rateLimitConfig;
 
-var workers = async.applyEachSeries([
-  worker.archiver(archiver_config),
-  worker.backwards_compatibility,
-  worker.remind_user_about_event(notifier_messager, mailroom),
-  worker.login_request(notifier_messager, mailroom),
-  worker.send_sms(notifier_messager),
-  worker.receive_coinbase_donation(notifier_messager, mailroom),
-  worker.reset_request(notifier_messager, mailroom),
-  worker.send_event_host_email(notifier_messager, mailroom),
-  worker.send_mofo_staff_email(notifier_messager, mailroom, process.env.MOFO_STAFF_EMAIL),
-  worker.send_new_user_email(notifier_messager, mailroom),
-  worker.event_mentor_confirmation_email(notifier_messager, mailroom),
-  worker.event_coorganizer_added(notifier_messager, mailroom),
-  worker.sign_up_for_bsd(notifier_messager),
-  worker.badge_awarded_send_email(notifier_messager),
-  worker.badge_application_denied(notifier_messager, mailroom),
-  worker.hive_badge_awarded(notifier_messager, mailroom),
-  worker.suggest_featured_resource(notifier_messager, process.env.SFR_SPREADSHEET, process.env.SFR_WORKSHEET)
-]);
+if ( process.env.DISABLE_RATE_LIMITING ) {
```

Comment:
I think we should remove the DISABLE_RATE_LIMITING option. It makes for a confusing start-up process flow. Lets just have a configurable TTL option as @davidascher suggested